augur/data/schema-subsample-config.json

```diff
@@ -87,7 +87,7 @@
                 "sequences_per_group": {
                     "type": "integer"
                 },
-                "subsample_max_sequences": {
+                "max_sequences": {
```

Comment:
Any opinions on 34e77af742007fcc9b77f74a10006ce8ab2c1a92? I'm leaning towards including it – having `subsample` in config names is a reflection of implementation detail which I am trying to hide everywhere else.

It would be good to decide now since we shouldn't make any config key name changes after release.

Reply:
> 'm leaning towards including it

👍 - using "subsample" within a subsampling's sample is... confusing

Reply:
Applied in [force-push](https://github.com/nextstrain/augur/compare/83731ade4f9298f26908065c51a2a03baa6d3eec..e3c3ae4538f3d8864e1b6153f106f384c24a4fb5).